src/main/java/hudson/plugins/cmvc/CmvcSCM.java

```diff
@@ -130,7 +133,7 @@
 	/**
 	 * Utility class
 	 */
-	private CommandLineUtil commandLineUtil = null;
+	private transient CommandLineUtil commandLineUtil = null;

```

Comment:
does this means that the field could have been serialized into a file (config?) and now it will not? what kind of effect will it have on the upgrade?

Reply:
the commandLineUtil variable is always built the same way the first time it's accessed (a singleton utility class). No need to serialize that.

Reply:
Perfect. Thanks. You can resolve both comments.